esp-wifi/src/wifi/mod.rs

```diff
@@ -427,8 +427,8 @@ static g_wifi_osi_funcs: wifi_osi_funcs_t = wifi_osi_funcs_t {
 
 const CONFIG_FEATURE_WPA3_SAE_BIT: u64 = 1 << 0;
 
-unsafe impl Sync for wifi_init_config_t {}
-unsafe impl Sync for wifi_osi_funcs_t {}
+// unsafe impl Sync for wifi_init_config_t {}
```

Comment:
Was this on purpose?

Reply:
They are commented out on purpose (they are implemented in the `esp-wifi-sys` package instead, now) but I meant to remove the comments.